docs/source/data-access-api/index.rst

```diff
@@ -16,21 +16,195 @@ that the full implementation of the functionality is yet to be completed.*
 
 The API can be accessed from the following URL [WIP]: https://api.dev.imap-mission.com
 
+Command Line Utility
+====================
+To Install
+----------
+
+Run the following command to use the API CLI.
+
+    .. code-block:: bash
+
+        pip install imap-data-access
+
+Base Command Arguments
+----------------------
+
+The following are base command arguments for the CLI:
+
+    .. code-block:: bash
+
+        imap-data-access -h # or
+        imap-data-access query # or
+        imap-data-access download # or
+        imap-data-access upload
+
+Add the -h flag with any base command for more information on use and functionality.
+
+.. openapi:: openapi.yml
+   :group:
+   :include: /query
```

Comment:
This part is describing the REST API specification, so it should be moved to under that section.